### ISSUE TITLE
module 'scipy' has no attribute 'stats'

### ISSUE THREAD

**Issue by hesmalek** [pauliacomi/pyGAPS#30]
**Describe the bug**

**To Reproduce**
- Code sample

import pygaps as pg

from scipy import stats, optimize, interpolate

isotherm = pg.isotherm_from_json(r'carbon_x1_n2.json')

isotherm.plot()

result_dict = pg.area_BET(isotherm, verbose=True)

- Error output

[module 'scipy' has no attribute 'stats']()

**Expected behavior**

**Screenshots**

**System (please complete the following information):**
 - OS: [e.g. Windows/Linux/MacOS]
 - Python Version [e.g. Python 3.6]
 - Toolchain [e.g. Conda, virtualenv]
I use Python 3.8.4 and pygaps 3.1.0 on windows 10. 

**Additional context**

Please run the quick start one time to check the errors with the updated consols.

**Where you are from?/How are you using pyGAPS? (optional)**
I'm always curious to see how pyGAPS is used and if it is useful to people. Don't fill this in if you don't want to tell me.

I'm from the UK. It's for personal use. I'm an adsorption researcher.

**Comment by pauliacomi** [comment]
Hi @hesmalek ,

Thanks for opening the issue.

Someone told me about this problem before. Currently putting the finishing touches on pyGAPS version 4.0, which should have this fixed. If you do not want to wait a couple of days until the release, you can install the dev branch like this:

```
pip install git+https://github.com/pauliacomi/pyGAPS@develop
``` 

The root cause was a lazy import for `scipy` I put in place to try to reduce the package load time. However it ended up messing up if you imported scipy normally alongside it. It was reverted in this commit: https://github.com/pauliacomi/pyGAPS/commit/034043ff94982742b1d35b3c1ab0996bf5bf6014

I will leave this issue open until the release comes out!

-pi

**Comment by pauliacomi** [comment]
Should be solved with the release of v4.0.2

Let me know if you have any further issues.

-pi

**Comment by hesmalek** [comment]
Thanks pi. It is solved now. Cheers. 

Hesmalek
